basic_pitch/inference.py

```diff
@@ -86,7 +86,13 @@ def get_audio_input(
 
     """
     assert overlap_len % 2 == 0, "overlap_length must be even, got {}".format(overlap_len)
-    audio_original, _ = librosa.load(audio_path, sr=AUDIO_SAMPLE_RATE, mono=True)
+
+    # audioread's mp3 reader does not support pathlib
+    if isinstance(audio_path, pathlib.Path) and audio_path.suffix == ".mp3":
+        audio_original, _ = librosa.load(str(audio_path), sr=AUDIO_SAMPLE_RATE, mono=True)
+    else:
+        audio_original, _ = librosa.load(audio_path, sr=AUDIO_SAMPLE_RATE, mono=True)
```

Comment:
```suggestion
    audio_original, _ = librosa.load(str(audio_path), sr=AUDIO_SAMPLE_RATE, mono=True)
```
Since librosa supports string paths regardless, it's probably easier to always use it